### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by Hassan0573** [community/community#62221] [New to GitHub]
### Select Topic Area

Question

### Body

How to contribute my github

**Comment by Imen-Frigui** [comment]
this will help you
https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Hassan0573 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. 

Check out this [post](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
